mobile_back_pixel/cpp/backend_tflite/tflite_pixel.cc

```diff
@@ -172,7 +172,7 @@ mlperf_backend_ptr_t mlperf_backend_create(
       if (strcmp(configs->accelerator, "gpu_f16") == 0)
         options.inference_priority1 = TFLITE_GPU_INFERENCE_PRIORITY_MIN_LATENCY;
       delegate = TfLiteGpuDelegateV2Create(&options);
-    } else if (strcmp(configs->accelerator, "nnapi") == 0) {
+    } else if (strcmp(configs->accelerator, "tpu") == 0) {
       auto options = tflite::StatefulNnApiDelegate::Options();
```

Comment:
I think it would be better to keep the `npu` here.
First of all, pixel backend still uses the `StatefulNnApiDelegate` option, same as tflite backend.

But also, Pixel backend is a copy of tflite with few modifications. I believe that it is regularly updated by copying a fresh version of tflite, and reapplying changes. I don't think we should add add unnecessary changes, it would make updating more error-prone.

Reply:
We should probably ask the backend maintainers from Google about this change.

Reply:
The accelerator is changed to `tpu` as requested here in this https://github.com/mlcommons/mobile_app_open/pull/579#issuecomment-1289856432

Reply:
@kcwu Are you ok with the change here?